CMakeLists.txt

```diff
@@ -5,7 +5,7 @@
 # and provide build configuration options.
 ######################################################################
 
-cmake_minimum_required(VERSION 2.8.12)
+cmake_minimum_required(VERSION 3.8)
```

Comment:
As well as erroring out if the CMake version's too low, this line effectively also calls `cmake_policy(SET CMPxxxx NEW)` for `CMP0024` to `CMP0067`. You'll therefore need to check each of those policies and ensure they don't have any unwanted effects, or switch to a different method of checking the minimum CMake version and enabling any new features you need.

Reply:
I'm saying it'll change the entire range, so 0024, 0025, 0026, ..., 0064, 0065, 0066 and 0067. That's forty-four policies to check, not two.

Reply:
Checked all policies, all looks good to have `NEW`